otel-infrastructure-collector/values.yaml

```diff
@@ -0,0 +1,118 @@
+global:
+  traces:
+    endpoint: "<traces-endpoint-here>"
+  metrics:
+    endpoint: "<metrics-endpoint-here>"
+  logs:
+    endpoint: "<logs-endpoint-here>"
+  defaultApplicationName: "default"
+  defaultSubsystemName: "nodes"
+
+opentelemetry-collector:
+  mode: deployment
+  fullnameOverride: otel-infrastructure-collector
+  clusterRole:
+    name: "otel-infrastructure-collector"
+    clusterRoleBinding:
+      name: "otel-infrastructure-collector"
+  replicaCount: 1
+  presets:
+    clusterMetrics:
+      enabled: true
+  ports:
+    otlp:
+      enabled: true
+    otlp-http:
+      enabled: false
+    jaeger-compact:
+      enabled: false
+    jaeger-thrift:
+      enabled: false
+    jaeger-grpc:
+      enabled: false
+    zipkin:
+      enabled: false
+
+  extraEnvs:
+  - name: CORALOGIX_PRIVATE_KEY
+    valueFrom:
+      secretKeyRef:
+        name: coralogix-keys
+        key: PRIVATE_KEY
+  config:
+    extensions:
+      zpages:
+        endpoint: localhost:55679
+    receivers:
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: opentelemetry-collector
+              scrape_interval: 30s
+              static_configs:
+                - targets:
+                    - ${MY_POD_IP}:8888
+    exporters:
+      coralogix:
+        timeout: "1m"
+        private_key: "${CORALOGIX_PRIVATE_KEY}"
+        traces:
+          endpoint: "{{ .Values.global.traces.endpoint }}"
+        metrics:
+          endpoint: "{{ .Values.global.metrics.endpoint }}"
+        logs:
+          endpoint: "{{ .Values.global.logs.endpoint }}"
+        application_name_attributes:
+        - "k8s.namespace.name" 
+        - "service.namespace"
+        subsystem_name_attributes:
+        - "k8s.deployment.name"
+        - "k8s.statefulset.name"
+        - "k8s.daemonset.name"
+        - "k8s.cronjob.name"
+        - "k8s.job.name"
+        - "k8s.container.name"
+        - "k8s.node.name"
+        - "service.name"
+        application_name: "{{.Values.global.defaultApplicationName }}"
+        subsystem_name: "{{.Values.global.defaultSubsystemName }}"
+    processors:
+      memory_limiter: null # Will get the k8s resource limits
+    service:
+      extensions:
+      - zpages
+      - health_check
+      - memory_ballast
+      pipelines:
+        metrics:
+          exporters:
+            - coralogix
+          processors:
+            - memory_limiter
+            - batch
+          receivers:
+            - prometheus
+            - otlp
+  tolerations: 
+    - operator: Exists
+
+  resources:
+    requests:
+      cpu: 100m
+      memory: 128Mi
+    limits:
+      cpu: 1
+      memory: 2G
+
+  # In order to enable podMonitor, following part must be enabled in order to expose the required port:
+  # ports:
+  #   metrics:
+  #     enabled: true
+
+  # podMonitor:
```

Comment:
When using the deployment mode, we need to use servicemonitor instead of podmonitor (https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/templates/servicemonitor.yaml#L1)